scripts/update-to-latest.sh

```diff
@@ -66,11 +66,15 @@ if ! git diff --no-ext-diff --quiet --exit-code; then
 	$HOME/bin/yarn run build-protocol-dts
 	$HOME/bin/yarn run changelog
 
-	# publish to npm
-	. $protocol_repo_path/scripts/publish-to-npm.sh "$commit_rev"
+	# bump npm version
+	npm version --no-git-tag-version "0.0.$commit_rev"
```

Comment:
**Note:** We still use `--no-git-tag-version` since we don’t want a separate Git commit, even though we do want a Git tag. (The CLI flag controls both commit + tag.)